src/libraries/System.Diagnostics.PerformanceCounter/tests/PerformanceCounterTests.cs

```diff
@@ -318,6 +320,32 @@ public static void PerformanceCounter_NextSample_MultiInstance()
             Helpers.DeleteCategory(categoryName);
         }
 
+        private static bool CanRunInInvariantMode => RemoteExecutor.IsSupported && !PlatformDetection.IsNetFramework;
```

Comment:
Is this still necessary? It was my understanding that `PlatformDetection.IsNetFramework` was not needed anymore (at least in .NET 5+). Is this assumption incorrect, @safern?

Reply:
This library is built for .NET and NetFX too. When running the tests, it will run twice, once against the .NET and once against NetFX. So this check still needed at least till now.